peg_src/jsdoctype.pegjs

```diff
@@ -194,11 +198,13 @@ InstanceMemberTypeOperator = "#" {
                            }
 
 
-BroadNamepathExpr = ExternalNameExpr
-                  / ModuleNameExpr
-                  / NamepathExpr
+BroadNamepathExpr =
+                  ExternalNameExpr / // no-typescript
+                  ModuleNameExpr / // no-typescript
```

Comment:
I think we need to keep `NamepathExpr`, and if so, then when the `no-typescript` removes the items, it will be left with a problematic initial slash. We might be able to move `NamepathExpr` to the front, but sometimes PEG and/or coverage would have problems with such rearranging, so I think it would be easiest to avoid tampering with it for now.

Reply:
Makes sense; I didn't know that pegjs was sensitive to whitespace.

Reply:
It isn't sensitive to whitespace, but because I have that file regex replace npm script which strips out the `no-typescript`, etc. lines, putting the slash at the beginning of a line could turn:

```pegjs
BroadNamepathExpr =
                  ExternalNameExpr // no-typescript
                  / ModuleNameExpr // no-typescript
                  / NamepathExpr
```

to:

```pegjs
BroadNamepathExpr =
                  / NamepathExpr
```

...where PegjS *does* have a problem with an initial slash like that.

Putting `NamepathExpr` first could work around this, but in some cases where I've moved an expression in a different order in PegJS (maybe this is what you were getting at as far as PegJS being sensitive), I've gotten something to the effect of left-hand parser recursion errors--something I should probably gain a fuller grasp of but am satisfied for now just knowing it can't always be done. :-)

In any case, thanks for the review and new version!